.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,47 @@
+---
+name: New Release
+about: Propose a new release
+title: Release v0.x.0
+assignees: marquiz
```

Comment:
Can we have multiple assignees? like Prow does?

Reply:
Yes, I suppose

Reply:
ping ^

Reply:
Added all (active) OWNERS, i.e. approvers and reviewers